### PR TITLE
OpenRTB 2.6-202409 + AdCom v1.0-202409

### PR DESCRIPTION
**Pull request by SyntaxNode**
- [OpenRTB 2.6-202409](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/releases/tag/2.6-202409)
- [AdCom v1.0-202409](https://github.com/InteractiveAdvertisingBureau/AdCOM/releases/tag/1.0-202409)

### CODE REVIEW

File: openrtb2/eid.go [hunk 1]
```diff
@@ -9,13 +9,53 @@ import "encoding/json"
...
+	Matcher string `json:"matcher,omitempty"`
+
+	// Attribute:
+	//   mm
+	// Type:
+	//   integer
+	// Description:
+	//   Match method used by the matcher. Refer to List: Delivery Methods
+	//   in AdCOM 1.0.
+	MM int64 `json:"mm,omitempty"`
```
Comment (SyntaxNode) [inline openrtb2/eid.go]: Implementing as omitempty instead of a pointer following our "zero value has no meaning" convention.

**Review by hhhjort — APPROVED** [review]
LGTM

File: openrtb2/eid.go [hunk 2]
```diff
@@ -9,13 +9,53 @@ import "encoding/json"
...
+	//
+	// May be omitted when mm=0, 1, or 2.
+	Matcher string `json:"matcher,omitempty"`
+
+	// Attribute:
+	//   mm
+	// Type:
+	//   integer
+	// Description:
+	//   Match method used by the matcher. Refer to List: Delivery Methods
```
Comment (bsardo) [inline openrtb2/eid.go]: Should this be `"Refer to List: ID Match Methods"`?

File: openrtb2/eid.go (hunk 1 above)
Comment (bsardo) [inline openrtb2/eid.go]: Should the type be `MatchMethod` instead of `int64`?

File: openrtb2/eid.go (hunk 2 above)
Comment (SyntaxNode) [reply openrtb2/eid.go]: Yes. Fixed.

File: openrtb2/eid.go (hunk 1 above)
Comment (SyntaxNode) [reply openrtb2/eid.go]: Yes. Fixed.

**Review by bsardo — APPROVED** [review]

**Pull request merged**